src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginEntry/FileOriginEntryViewModel.cs

```diff
@@ -9,33 +16,60 @@ namespace NexusMods.App.UI.Pages.ModLibrary.FileOriginEntry;
 
 public class FileOriginEntryViewModel : AViewModel<IFileOriginEntryViewModel>, IFileOriginEntryViewModel
 {
-    public required string Name { get; init; } 
-    public string Version { get; init; } = "-";
-    public required Size Size { get; init; }
-    public DateTime ArchiveDate { get; set; } = DateTime.UnixEpoch;
-
-    [ObservableAsProperty]
-    public string DisplayArchiveDate { get; } = "-";
-
-    
-    [Reactive]
-    public DateTime LastInstalledDate { get; set; } = DateTime.UnixEpoch;
-    
-    [ObservableAsProperty]
-    public string DisplayLastInstalledDate { get; } = "-";
-    public required ReactiveCommand<Unit, Unit> AddToLoadoutCommand { get; init; }
-    
-    public FileOriginEntryViewModel()
+    public string Name { get; }
+    public string Version { get; }
+    public Size Size { get; }
+    public DateTime ArchiveDate { get; }
+    public ReactiveCommand<Unit, Unit> AddToLoadoutCommand { get; init; }
+
+    [ObservableAsProperty] public string DisplayArchiveDate { get; } = "-";
+
+    [ObservableAsProperty] public DateTime LastInstalledDate { get; set; }
+
+    [ObservableAsProperty] public string DisplayLastInstalledDate { get; } = "-";
```

Comment:
I really hate this attribute. It doesn't do what you expect it to do. Reflection is used at runtime to assign to the backing field. This attribute is pure evil, and shouldn't be used.